src/webview/serverless-function/app/createFunction.tsx

```diff
@@ -0,0 +1,332 @@
+            || !this.state.wsFolderPath || this.state.language.length === 0 || this.state.template.length === 0;
+    }
+
+    createFunction = (): void => {
+        this.props.onCreateSubmit(this.state.functionData.name,
+            this.convert(this.state.language), this.convert(this.state.template), this.state.wsFolderPath, this.state.imageData.name)
+    }
+
+    render(): React.ReactNode {
+        const { wsFolderItems, wsFolderPath, functionData, templates, language, template, images, imageData } = this.state;
+        const languages = ['Go', 'Node', 'Python', 'Quarkus', 'Rust', 'Spring Boot', 'TypeScript'];
+        const folders: any[] = ['New Folder'];
+        folders.push(...wsFolderItems);
+        const filter = createFilterOptions<string>();
+        const imageRegex = RegExp('[^/]+\\.[^/.]+\\/([^/.]+)(?:\\/[\\w\\s._-]*([\\w\\s._-]))*(?::[a-z0-9\\.-]+)?$');
+        return (
+            <>
+                <LoadModal show={this.props.loadScreen} />
+                <FormControl sx={{ margin: '2rem 0 0 2rem' }}>
+                    <TextField
+                        type='string'
+                        variant='outlined'
+                        required
+                        defaultValue={functionData.name}
+                        error={functionData.error}
+                        onChange={(e) => this.validateName(e.target.value)}
+                        id='function-name'
+                        placeholder='Function Name *'
+                        sx={{
+                            input: {
+                                color: 'var(--vscode-settings-textInputForeground)',
+                                height: '5px !important',
+                                '&::placeholder': {
+                                    color: 'var(--vscode-settings-textInputForeground) !important',
+                                    opacity: '1 !important'
+                                }
+                            }
+                        }}
+                        helperText={functionData.helpText}
+                    />
+                </FormControl>
+                <FormControl sx={{ margin: '2rem 0 0 2rem', width: 'auto' }}>
+                    <Autocomplete
+                        value={imageData.name}
+                        disabled={functionData.error || functionData.name.length === 0}
+                        onChange={(_event, newValue: string) => {
+                            if (newValue) {
+                                const value = newValue.replace('Add', '').trim();
+                                if (!imageRegex.test(value)) {
+                                    this.setState({
+                                        imageData: {
+                                            name: '',
+                                            error: true,
+                                            helpText: 'Provide full image name in the form [registry]/[namespace]/[name]:[tag]'
```

Comment:
Where are we using this helptext. I see the drop down automatically fetched quay and docker.io image names, which can be completely incorrect.

Reply:
When user entered invalid image name then help text will appear
![help-text](https://github.com/redhat-developer/vscode-openshift-tools/assets/93245779/39ca39c9-8f1d-4126-be16-78b503f18728)